StPicoEvent/StPicoBTofPidTraits.cxx

```diff
@@ -1,3 +1,13 @@
+/***************************************************************************
+ *
+ * $Id$
+ *
+ * Author: Xin Dong (xdong@lbl.gov)
```

Comment:
I also agree that the commits history to know the contributors. README file is fine too. 

I added this because it is required in the guidelines. I will be more than happy to skip it if you reviewers ignore it ;-)